providers/google/pyproject.toml

```diff
@@ -143,7 +143,8 @@ dependencies = [
 # Any change in the dependencies is preserved when the file is regenerated
 [project.optional-dependencies]
 "apache.beam" = [
-    "apache-beam[gcp]",
+    'apache-beam[gcp]>=2.53.0; python_version < "3.12"',
```

Comment:
One of the tests started failing when we isolated apache-beam and it turned out that we did not have the same limits set here as in apache.beam provider. That's another side-effect of better isolation between providers.